doc/README.yaml.md

```diff
@@ -0,0 +1,170 @@
+
+# README for Open-CE Environment Files
+
+
+This README documents the Open-CE Environment YAML files used by the build scripts
+[build_env.py](README.build_env.md)
+and [build_feedstock.py](README.build_feedstock.md).
+Please refer to the respective README files for each of these two scripts.
+
+Most likely you will want to simply use the Open-CE environment files provided within the
+open-ce repository as they are. These files have already been customized and
+verified to work. However, if you want to debug or do your own customizations,
+you can edit them or create your own environment files for use by these scripts. If you
+do this, you will likely want to start by copying one of the existing files and
+making local edits as desired.
+
+
+# Open-CE Environment File
+
+The `build_env.py` command uses what we call an Open-CE environment file.
+In this context, an "environment" is a fully-built core package along with all
+of its dependencies, each of which is found in a separate "feedstock" repository
+within the broader open-ce project.
+Default environment files are provided for each of several environments in the open-ce
+repository, within [https://github.com/open-ce/open-ce/envs](../envs)
```

Comment:
https://github.com/open-ce/open-ce/envs -> This link isn't right. It should be https://github.com/open-ce/open-ce/tree/master/envs.

Reply:
I just viewed this README from your branch. The links are correct but pointing to your branch. I think they will be updated once this is merged to master branch.